ext/liquid_c/document_body.h

```diff
@@ -28,6 +28,14 @@ typedef struct document_body {
     c_buffer_t buffer;
 } document_body_t;
 
+typedef struct document_body_header {
+    uint32_t entrypoint_block_index;
```

Comment:
I see it's called index here, but it ends up being used as an offset here: https://github.com/Shopify/liquid-c/pull/138/files#diff-d25b9c9a1b16f1935c261f8232ea43e790513f022f33b7c8de117016cd76f130R188 correct?

Should this be renamed to _offset or the code is wrong in #138?

Reply:
You're right, it is an offset.  There isn't currently a block table to index into.